### PR TITLE
Various software list kana metadata cleanups.

### PR DESCRIPTION
**Pull request by 0kmg**
- Replaced halfwidth characters (including middle dot) with fullwidth ones.
- Fixed a few alt titles.

### CODE REVIEW

**Comment by 0kmg** [comment]
Halfwidth stuff not replaced:

fmtowns_cd.xml: uses halfwidth middle dots as ellipsis in two spots.
fmtowns_flop_orig.xml: one instance that seems to be quoting something.
mz2500.xml: one original filename.
pc8801_flop.xml: 77 lines, mostly quotes of on screen menu stuff?

**Comment by 0kmg** [comment]
smc777 alt titles come from the actual labels. The baseball game really does use kanji in game, though MAME doesn't get that far.

I don't know why SMS's Sukeban Deka II alt title had the デカ in front. That's just the reading for 刑事 for this title. I suppose it's an ancient copy/paste from somebody how didn't know how kanji readings are given (furigana).

**Comment by DopefishJustin** [comment]
Yeah it's clearly a case of having read the furigana as the first line and the title as a second line

File: hash/pico.xml [hunk 1]
```diff
@@ -4928,7 +4928,7 @@ But how do later protos fit with this theory? Maybe the later protos were from t
 		<publisher>Sega Toys</publisher>
 		<info name="serial" value="HPC-6114"/>
 		<info name="release" value="200207xx"/>
-		<info name="alt_title" value="トミカ・ピコ レスキューパーキング きんきゅうしゅつどう! はたらくのりもの &lt;初回･トミカ付属&gt; - Tomica Pico Rescue Parking Kinkyuushutsudou! Hataraku Norimono ~Shokai-Tomica Fuzoku~"/>
+		<info name="alt_title" value="トミカ・ピコ レスキューパーキング きんきゅうしゅつどう! はたらくのりもの &lt;初回・トミカ付属&gt; - Tomica Pico Rescue Parking Kinkyuushutsudou! Hataraku Norimono ~Shokai-Tomica Fuzoku~"/>
```
Comment (cuavas) [inline hash/pico.xml:4931]: Why is the transliteration in the description as well as being appended to the Japanese title here?  I don’t think it needs to be in both.

Comment (0kmg) [reply hash/pico.xml:4931]: No, it isn't on the box (as one would expect): https://i.ebayimg.com/images/g/5HIAAOSw3J1h~C3x/s-l1600.jpg

**Pull request merged**